Testreglar/2.1.1/Nett/nett-2.1.1a.json

```diff
@@ -0,0 +1,173 @@
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Får du til å betene alle lenker, knappar og funksjonar med tastaturet, utan tidsberegning av dei enkelte tastetrykka?",
+			"ht": "<p>Trykk sakte på tab-tasten for å navigere på nettsida med tastaturet. </p>\n<ul>\n<li>Følg nøye med på navigasjonen og informasjonen i statuslinja i nettlesaren </li>\n<li>For å betene innhaldet kan du også bruke enter, mellom og pil.</li>\n<li>For å rygge med tastaturet, bruk Shift+Tab.</li>\n<li>Det kan vere at du må opne menyar og modaler for å teste alt innhald.</li>\n<li>For å sjekke om det er mogleg å betene lenker, kan du sjå om URL står statuslinja i nettlesaren. Det er ikkje nødvendig å aktivere lenka.</li>\n</ul>\n<p>Eksempel på tidsberegning er at brukaren må: </p>\n<ul>\n<li>utføre eller gjenta mange tastetrykk i løpet av ein kort periode.</li>\n<li>halde nede en tast i ein lengre periode før tastetrykket blir registret. </li>\n</ul>\n<p><strong>Merk:</strong> Dersom det er funksjonalitet som kan betenast på fleire måtar, er det tilstrekkeleg at ein av dei kan brukast med tastaturet. Eksempel på dette er datoveljar, kor brukaren både kan skrive inn dato, og velje dato frå ein kalendar.</p>\n<p><strong>Merk:</strong> Eventuell tastaturfelle, omlasting av sida eller kontekstendring påvirkar ikkje om det mogleg å betene innhaldet.</p>\n<p><strong>Merk: </strong>Handlingar som blir utført med mus, skal og kunne bli gjort med tastaturet. Eksempel på handlingar er å klikke, velge, flytte og forstørre.</p>",
+			"type": "jaNei",
+			"kilde": [
+				"G202"
+			],
+			"ruting": {
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"ja": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Det er mogleg å nå og betene alt innhald/funksjonalitet på nettsida med tastatur."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Beskriv elementet du ikkje nådde eller fekk til å betene med tastaturet",
+			"ht": "",
+			"type": "tekst",
+			"label": "Element:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Er ein alternativ måte å navigere på, dokumentert i nærleiken av det aktuelle elementet?",
+			"ht": "<p>Dersom det er behov for å bruka noko anna enn standard pil- eller tabulatortaster eller andre standardmetoder for tastaturnavigering, får brukeren informasjon om metode som må brukast for å nå elementet.</p>\n<p><strong>Merk: </strong>Dette bygger på analogisk bruk av dokumentasjonskravet som står i ordlyden i 2.1.2 Ingen tastaturfelle.</p>\n<p><strong>Merk: </strong>Alternativ fremgangsmåte skal vere beskrevet i nærheten av det aktuelle elementet.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Det er mogleg å nå og betene all funksjonalitet på nettsida via dokumentert, alternativ tastaturnavigasjon."
```

Comment:
Du kan vell ikke garantere at du kan **nå og betene all funksjonalitet på nettsida** på dette steget.  Dersom du har to element du ikke når med tastaturet og bare én av det har et alternativ vil dette utfallet bli feil. Må heller være feks "Innhold det er mogleg å nå og betene via dokumentert, alternativ tastaturnavigasjon."